src/applications/disability-benefits/all-claims/pages/privateMedicalRecordsRelease.js

```diff
@@ -0,0 +1,151 @@
+import _ from '../../../../platform/utilities/data';
```

Comment:
Is this a pattern we've used before? This is the first time I've seen it. Can't decide if it's cool or misleading.

Reply:
I've definitely seen this used in other places. Not sure how common.